grpc/channelz/v1/channelz.proto

```diff
@@ -117,10 +117,42 @@ message ChannelData {
   google.protobuf.Timestamp last_call_started_timestamp = 7;
 }
 
+// A trace event is an interesting thing that happened to a channel or
+// subchannel, such as creation, address resolution, subchannel creation, etc.
+message ChannelTraceEvent {
+  // High level description of the event.
+  string description = 1;
+  // The supported severity levels of trace events.
+  enum Severity {
```

Comment:
Can you prefix these with `CT_`? I had to do that to get C++ proto to compile, and I forgot to make the change in my design

Reply:
```
  enum Severity {
    CT_UNKNOWN = 0;
    CT_INFO = 1;
    CT_WARNING = 2;
    CT_ERROR = 3;
  }
```

Reply:
This is the proto that is being used in C++ test; [here](https://github.com/grpc/grpc/blob/master/src/proto/grpc/channelz/channelz.proto). I have been meaning to write a linter to make sure it stays aligned with this one

Reply:
Done.